Add openshift profile to allow openshift deployment (#15)

cunningt opened the pull request:
Cherry pick openshift deployment changes to 4.0.0-branch

cunningt committed:
Add openshift profile to allow openshift deployment (#15)

mcarlett commented on spring-boot/sap-idoc-destination-spring-boot/pom.xml:
```diff
@@ -109,8 +138,73 @@
+								<image>
+									<name>camel/actuator-http-metric</name>
+									<build>
+										<from>quay.io/jkube/jkube-java:0.0.19</from>
```
can this image be parameterized? it would be useful in case we want to test different images, thanks

mcarlett reviewed (commented):
Can you please update the readme file in order to provide correct deploy steps? thanks!

cunningt committed:
Change image name to match artifact name, add openshift deployment instructions, paramaterize the base image name

cunningt commented:
> Can you please update the readme file in order to provide correct deploy steps? thanks!

Added deploy steps, paramaterized base image, changed image names to match the artifact.
